apps/qubit/modules/term/templates/deleteSuccess.php

```diff
@@ -30,7 +30,7 @@
         </div>
       <?php endif; ?>
 
-      <?php if (0 < count($resource->descendants)): ?>
+      <?php if (0 < ($resource->rgt - $resource->lft - 1) / 2): ?>
```

Comment:
I think `<?php if (0 < ($resource->rgt - $resource->lft - 1)): ?>` (without dividing by 2) is necessary and sufficient to test for descendants here.

Reply:
Good point @djjuhasz, I was going to address that and noticed that the actual count is shown just below, so I've moved the calculation to the action and used it for both cases (like it's done on IO delete action and template).

Reply:
Sounds good @jraddaoui.  I can't see your change to the calculation in the Github UI, I'm not sure why. :(

Reply:
https://github.com/artefactual/atom/pull/1022/commits/e5fb1283dd99128a589d683b6318b09633efb3e6